tests/jobs/test_scheduler_job.py

```diff
@@ -1242,12 +1165,12 @@ def test_execute_task_instances_unlimited(self):
         date = dag.start_date
         tis = []
         for _ in range(0, 20):
+            date = dag.following_schedule(date)
             dr = dag.create_dagrun(
                 run_type=DagRunType.SCHEDULED,
                 execution_date=date,
                 state=State.RUNNING,
             )
-            date = dag.following_schedule(date)
```

Comment:
Hmm, I think this changes the behaviour…?

Reply:
Not really. What’s happening is that it’s now skipping creation of the first dagrun because the dag_maker fixture has created it. Without this, it’ll fail on integrity error while trying to create the same dagrun already created by fixture

Reply:
I see. Thanks.